build.sbt

```diff
@@ -1,68 +1,17 @@
-lazy val root = (project in file("."))
-  .settings(mainClass in Universal := Some("org.scalaexercises.evaluator.EvaluatorServer"))
-  .settings(stage <<= (stage in Universal in `evaluator-server`))
-  .settings(noPublishSettings: _*)
-  .aggregate(
-    `evaluator-server`,
-    `evaluator-shared-jvm`,
-    `evaluator-shared-js`,
-    `evaluator-client-jvm`,
-    `evaluator-client-js`)
-
-lazy val `evaluator-shared` = (crossProject in file("shared"))
+lazy val `evaluator-shared` = (project in file("shared"))
```

Comment:
Do we need this shared module? I'm guessing that we can convert this project in a mono-module one (everything under root)

Reply:
I can move types.scala in server project, but smoketests is an independent project.

Reply:
Sounds good, we could have those two modules: server and smoketests 👍